pkg/anonymization/anonymizer.go

```diff
@@ -231,6 +246,45 @@ func (anonymizer *Anonymizer) ObfuscateIP(ipStr string) string {
 	return "::"
 }
 
+// Stores the translation table in a Secret in the openshift-insights namespace.
+// The actual data is stored in the StringData protion of the Secret.
+// Uses the Apply mechanism so it creates the secret if not present or updates it when present.
+func (anonymizer *Anonymizer) StoreTranslationTable() *corev1.Secret {
+	if len(anonymizer.translationTable) == 0 {
+		return nil
+	}
+	defer anonymizer.ResetTranslationTable()
+
+	secret := &applycorev1.SecretApplyConfiguration{
+		TypeMetaApplyConfiguration: applymetav1.TypeMetaApplyConfiguration{
+			Kind:       &secretKind,
+			APIVersion: &secretAPIVersion,
```

Comment:
Is this required pls? It doesn't seem right to hardcode the API version.....

Reply:
Yes
```
Failed to create/update the translation table secret. err: Incorrect version specified in apply patch. Specified patch version: , expected: v1
```

We already hardcode API version in the `mainfests`, and IMO we should just update the API version on the fly